README.md

```diff
@@ -1,545 +1,132 @@
+  </a>
+  <a href="https://twitter.com/intent/follow?screen_name=stratumv2">
+    <img src="https://img.shields.io/twitter/follow/stratumv2?style=social" alt="X (formerly Twitter) Follow">
+  </a>
+</p>
 
-## MSRV
-Minimum Supported Rust Version: 1.75.0
+## 💼 Table of Contents
 
-## Table of Contents
-
-- [Stratum](#stratum)
-  - [Table of Contents](#table-of-contents)
-  - [1. Goals](#1-goals)
-  - [2. Common Use Cases](#2-common-use-cases)
-  - [2.1 Miners](#21-miners)
-  - [2.2 Pools](#22-pools)
-  - [3. Structure](#3-structure)
-    - [3.01 protocols](#301-protocols)
-    - [3.02 protocols/v1](#302-protocolsv1)
-    - [3.03 protocols/v2](#303-protocolsv2)
-    - [3.04 protocols/v2/const-sv2](#304-protocolsv2const-sv2)
-    - [3.05 protocols/v2/binary-sv2](#305-protocolsv2binary-sv2)
-    - [3.06 protocols/v2/framing-sv2](#306-protocolsv2framing-sv2)
-    - [3.07 protocols/v2/codec-sv2](#307-protocolsv2codec-sv2)
-    - [3.08 protocols/v2/subprotocols](#308-protocolsv2subprotocols)
-    - [3.09 protocols/v2/sv2-ffi](#309-protocolsv2sv2-ffi)
-    - [3.10 protocols/v2/noise-sv2](#310-protocolsv2noise-sv2)
-    - [3.11 protocols/v2/roles-logic-sv2](#311-protocolsv2roles-logic-sv2)
-    - [3.12 utils/buffer](#312-utilsbuffer)
-    - [3.13 utils/network-helpers](#313-utilsnetwork-helpers)
-    - [3.14 roles/mining-proxy](#314-rolesmining-proxy)
-    - [3.15 roles/pool](#315-rolespool)
-    - [3.16 roles/test-utils/pool](#316-rolestest-utilspool)
-    - [3.17 roles/test-utils/mining-device](#317-rolestest-utilsmining-device)
-    - [3.18 examples](#318-examples)
-    - [3.19 examples/interop-cpp](#319-examplesinterop-cpp)
-    - [3.20 examples/interop-cpp-no-cargo](#320-examplesinterop-cpp-no-cargo)
-    - [3.21 examples/ping-pong-with-noise](#321-examplesping-pong-with-noise)
-    - [3.22 examples/ping-pong-without-noise](#322-examplesping-pong-without-noise)
-    - [3.23 examples/sv1-client-and-server](#323-examplessv1-client-and-server)
-    - [3.24 examples/sv2-proxy](#324-examplessv2-proxy)
-    - [3.25 examples/template-provider-test](#325-examplestemplate-provider-test)
-    - [3.26 test](#326-test)
-    - [3.27 experimental](#327-experimental)
-  - [4. Build/Run/Test](#4-buildruntest)
-    - [4.01 Build](#401-build)
-      - [Building/Updating sv2-ffi](#buildingupdating-sv2-ffi)
-    - [4.02 Test](#402-test)
-    - [4.03 Run](#403-run)
-  - [5. Branches](#5-branches)
-  - [6. Logging](#6-logging)
-  - [7. proxy-config.toml file](#7-proxy-configtoml-file)
-
-## 1. Goals
+<p align="center">
+  <a href="#introduction">Introduction</a> •
+  <a href="#getting-started">Getting Started</a> •
+  <a href="#use-cases">Use Cases</a> •
+  <a href="#roadmap">Roadmap</a> •
+  <a href="#contribute">Contribute</a> •
+  <a href="#support">Support</a> •
+  <a href="#donate">Donate</a> •
+  <a href="#supporters">Supporters</a> •
+  <a href="#license">License</a> 
+  <a href="msrv">MSRV</a>
```

Comment:
I think here one '#' is missing (?)
The link is not working on your fork

Reply:
Good catch, I've fixed this